connexion/apps/aiohttp_app.py

```diff
@@ -90,6 +90,6 @@ def run(self, port=None, server=None, debug=None, host=None, **options):
             if options.get('use_default_access_log'):
```

Comment:
```suggestion
            if options.pop('use_default_access_log'):
```
This is a connexion-specific option, so we should pop it here and not pass it on the aiohttp.